packages/connector-google/README.md

```diff
@@ -1,7 +1,33 @@
-# Google
+# Google connector
 
 The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system.
 
+Google 连接器为你的应用程序提供了一种接入 Google OAuth 2.0 身份验证系统的简单方式。
+
+**Table of contents**
+- [Google connector](#google-connector)
+  - [Set up a project in the Google API Console](#set-up-a-project-in-the-google-api-console)
+  - [Configure your consent screen](#configure-your-consent-screen)
+    - [Configure and register your application](#configure-and-register-your-application)
+    - [Edit app registration](#edit-app-registration)
+      - [Config OAuth consent screen](#config-oauth-consent-screen)
+      - [Config scopes](#config-scopes)
+      - [Add test users (External user type only)](#add-test-users-external-user-type-only)
+  - [Obtain OAuth 2.0 credentials](#obtain-oauth-20-credentials)
+    - [Config types](#config-types)
+  - [References](#references)
+- [Google 链接器](#google-链接器)
+  - [在 Google API 控制台中创建一个项目](#在-google-api-控制台中创建一个项目)
+  - [配置 OAuth 同意屏幕](#配置-oauth-同意屏幕)
+    - [注册和配置应用](#注册和配置应用)
+    - [修改应用注册](#修改应用注册)
+      - [设置「OAuth 同意屏幕」](#设置oauth-同意屏幕)
+      - [配置权限「范围」](#配置权限范围)
+      - [添加「测试用户」（仅「User Type」设置为「外部」时需要）](#添加测试用户仅user-type设置为外部时需要)
```

Comment:
Oh, shoot. I have to trim these special characters in AC guide to support these internal toc links